packages/calimero-sdk/src/login/refreshToken.ts

```diff
@@ -0,0 +1,22 @@
+import apiClient from '../api';
+import { JwtTokenResponse } from '../api/nodeApi';
+import { setAccessToken, setRefreshToken } from '../storage';
+import { ResponseData } from '../types';
+
+interface GetNewJwtTokenProps {
+  refreshToken: string;
+  getNodeUrl: () => string;
+}
+
+export const getNewJwtToken = async ({
+  refreshToken,
+  getNodeUrl,
+}: GetNewJwtTokenProps) => {
+  const tokenResponse: ResponseData<JwtTokenResponse> = await apiClient
+    .node()
+    .refreshToken(refreshToken, getNodeUrl());
+  if (tokenResponse.data) {
```

Comment:
How do you handle error?

Reply:
Added proper error handling within this function and handler on client side.